Source/Core/DolphinWX/Main.cpp

```diff
@@ -108,8 +107,16 @@ bool DolphinApp::OnInit()
   wxHandleFatalExceptions(true);
 #endif
 
+#ifdef _WIN32
+  AttachConsole(ATTACH_PARENT_PROCESS);
```

Comment:
Not really. We should probably `FreeConsole` at some point before quitting if this worked.